docs/operate/reference/services/discovery/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Discovery Service"
+linkTitle: "Discovery"
+description: "The discovery service is used to discover resources on a machine."
+type: docs
+weight: 50
+no_list: true
+#icon: true
+#images: ["/services/icons/discovery.svg"]
+tags: ["discovery", "services"]
+no_service: true
+date: "2025-03-03"
+# updated: ""  # When the content was last entirely checked
+# SMEs: John N.
+modulescript: true
+---
+
+The discovery service is used to discover resources on a machine.
+If you are [creating a modular resource](/operate/get-started/other-hardware/) that depends on other resources that are discoverable in a systematic way, you can create a discovery service as part of your module to discover those resources.
+
+## Example usage
+
+Imagine you are creating a vision service module that depends on a camera.
+You might want to include both a vision service and a discovery service in your module.
+You could write a discovery service that reports all camera paths recognized by your computer or SBC.
+Users of your module could then:
+
+1. Configure the discovery service in their machine's configuration.
+1. Use it to discover any cameras that are connected to the machine.
+1. Use one of the discovered cameras in their vision service configuration.
```

Comment:
[q] so visually it looks like this works, but I notice in some examples we step through numbers ourselves(ex 1. 2. 3.) is it preferred to leave it like this?

Reply:
These render correctly and I like doing it this way since when you add or delete a step you don’t have to change them all. Thanks for checking!